angularSTEP/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-toolbar color="primary" class="mat-elevation-z8">
+    <span>Recipe Stuff</span>
+
+    <a mat-button>Home</a>
+    <a mat-button>Discover</a>
+    <a mat-button>Recipes</a>
```

Comment:
I would change these to buttons. 
```suggestion
    <button mat-button>Home</button>
    <button mat-button>Discover</button>
    <button mat-button>Recipes</button>
```